platforms/web/lib/useWysiwyg.formatting.test.tsx

```diff
@@ -94,25 +133,21 @@ describe.each([
             expect(button).toHaveAttribute('data-state', defaultState);
         });
 
-        it(
-            `Should be ${expectedActivationState} ` + `after single activation`,
-            async () => {
-                // When
-                await act(() => userEvent.click(button));
-
-                // Then
-                await waitFor(() =>
-                    expect(button).toHaveAttribute(
-                        'data-state',
-                        expectedActivationState,
-                    ),
-                );
-            },
-        );
+        // eslint-disable-next-line max-len
+        it(`Should be ${expectedActivationState} after single activation`, async () => {
+            // When
+            await userEvent.click(button);
+
+            // Then
+            expect(button).toHaveAttribute(
+                'data-state',
+                expectedActivationState,
+            );
+        });
```

Comment:
There were quite a few unnecessary acts and await/waitFor's so I removed as many as I could